apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOptimize/index.tsx

```diff
@@ -484,9 +487,13 @@ export default function TabBuild() {
             </Trans>
           </MenuItem>)}
         </DropdownButton>
-        <Button sx={{ borderRadius: "4px 0px 0px 4px" }} color='warning' onClick={() => setnotification(n => !n)} >
-          {notification ? <NotificationsActiveIcon /> : <NotificationsOffIcon />}
-        </Button>
+        <BootstrapTooltip placement="top" title={t`notifyTooltip`}>
+          <span>
+            <Button sx={{ borderRadius: 0 }} color='warning' onClick={() => setnotification(n => !n)} >
```

Comment:
Something messed up 1px on the bottom reee
![image](https://user-images.githubusercontent.com/36019388/216234435-ae4afd2e-ba7b-46a5-90fc-153f34d6a9af.png)

Reply:
MADGE its always gonna be off its a rounding issue... best I can do is this
<img width="600" alt="image" src="https://user-images.githubusercontent.com/1754901/216238365-482ed59d-3cf1-4350-beb7-da5a0bb5eb1e.png">

Reply:
But it was fine before the 2nd most recent commit

Reply:
yeah because now its wrapped in a tooltip and a `span`